ME3TweaksCore/Localization/Dictionaries/int.xaml

```diff
@@ -39,7 +39,7 @@
     <system:String x:Key="string_interp_cannotBackupGameIsRunning">Cannot backup while {0} is running.</system:String>
     <!-- 0={game name (mass effect, mass effect 2...)} -->
     <system:String x:Key="string_ensureCorrectGameChosen">Ensure correct game chosen</system:String>
-    <system:String x:Key="string_warningLinkedTargetWillNotLoad">The path you specify will be checked if it is a vanilla backup. Once this check is complete it will be marked as a backup and modding tools will refuse to modify it. Ensure this is not your active game path or you will be unable to mod the game.</system:String>
+    <system:String x:Key="string_warningLinkedTargetWillNotLoad">It will be checked if the path you specify is a vanilla backup. Once this check is complete it will be marked as a backup and modding tools will refuse to modify it. Ensure this is not your active game path or you will be unable to mod the game.</system:String>
```

Comment:
I don't think this change works - the message here is in a dialog. It is 

Ensure correct game chosen
The path you specify...

Due to how the title and dialog portions are split it's not really a continuation or a sentence.